lms/djangoapps/grades/tests/test_new.py

```diff
@@ -112,17 +112,36 @@ class SubsectionGradeFactoryTest(GradeTestBase):
     enable saving subsection grades blocks/enables that feature as expected.
     """
 
+    def assert_grade(self, grade, expected_earned, expected_possible):
+        """
+        Asserts that the given grade object has the expected score.
+        """
+        self.assertEqual(
+            (grade.all_total.earned, grade.all_total.possible),
+            (expected_earned, expected_possible),
+        )
+
     def test_create(self):
```

Comment:
**Note to reviewers:** Moved `SubsectionGradeTest.test_compute` to `SubsectionGradeFactoryTest.test_create` since that's what it really is.